tests/Aspire.Azure.AI.OpenAI.Tests/AspireAzureAIOpenAIExtensionsTests.cs

```diff
@@ -65,4 +65,32 @@ public void ConnectionStringCanBeSetInCode(bool useKeyed)
 
         Assert.NotNull(client);
     }
+
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public void ReadsUriFromConnectionStringsCorrectly(bool useKeyed)
+    {
+        var builder = Host.CreateEmptyApplicationBuilder(null);
+        builder.Configuration.AddInMemoryCollection([
+            new KeyValuePair<string, string?>("ConnectionStrings:openai", "https://yourservice.openai.azure.com/")
```

Comment:
I would add another test with `host:port` just to be sure ;-) -- this was the URI I was using was http://foo:12345

Reply:
Done